collections/_development_standards/new-solutions.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+title:  "New Solutions"
+category: development
+---
+
+## New solutions
+
+Below are some considerations that we should be taking into account when creating new solutions.
+
+### Which .Net version should I use
+
+In the main new solutions should be using the latest long-term support version available at the time. It also needs to be supported by our build servers so a check-in with the DevOps team before starting is always useful.
+
+### Storage mechanisms
+
+Azure SQL or CosmosDB, Is the data likely to be structured or unstructured, will it require fast reads and horizontal scalability?
+How will we extract the data out into the DataMart solution?
+
+### Analytics
+
+Do we need to and how will we apply analytics to the user journey's and how do we do that?
+
+### Code quality
+
+How will you ensure and prove the quality of the code when it comes to handing over to live running?
+The use of SonarCloud can be of use here as it is a way to visually show that the code you're handing over has a level of quality
+
+
+### Authentication mechanisms
+
+There are several authentication mechanisms in-use across the service, these are usually split down by user-group.
+
+|---|---|---|
+|Staff IDAMS|Used for internal staff authentication|For instance internal support applications would use this mechanism|
+|Provider IDAMS|Used to authentication training provider users||
+|Employer User|Used to authentication employer users||
+
+### Interoperability with other solutions
+
+It is good to be aware that some solutions may be on older versions of .Netcore or may still be on .Net Framework. This may add work for you to upgrade or conditionally compile the older dependencies. This can be especially prevalent with the use of API Client Nuget packages.
+
+### Is that sparkly new technology worth the learning curve?
+
+We all love new tech but is it too new to have decent support, is there an additional cost involved, do we have the skills to support it once its been built?
```

Comment:
Any new technologies should be raised across teams (Arch, Dev, DevOps, Test) and agreed before its adopted. We'd expect this to occur during Alpha and be proven to be viable. Important that this process is followed (i.e. Can't introduce new tech without these experiments).

Reply:
@DavidBSteele Sounds like this is already an established process? What wording would you suggest to align this with that process?

Reply:
I think we need to document the process. Its an understood process but not well documented in our GitPages. Its on the list of things that Roy and I are going to process but its slow going.

We could put something like this for the time being (and then link to more detail in the future?)...
"Any new technologies that a team wishes to use must be raised across teams (Arch, Dev, DevOps, Test) and agreed before its adopted. This process normally occurs during the alpha phase of a project. Its key that before a new technology is introduced into a Live product that it is proven to be viable. Its vital that this process is followed (i.e. Can't introduce new tech without these experiments) to prevent investment in unsupportable technologies."

Reply:
I've updated the wording